app/models/invoice.rb

```diff
@@ -1,2 +1,42 @@
 class Invoice < ApplicationRecord
+
+  def self.find_by_params(param)
```

Comment:
@Laszlo-JFLMTCO @akintner i believe if you use strong params in your controller, you don't need to write this method. you could just pass your params to the `find_by` method that's coming from ActiveRecord